pkgs/tools/security/step-ca/default.nix

```
@@ -1,19 +1,30 @@
-{ lib, buildGoPackage, fetchFromGitHub }:
+{ lib, stdenv, pkgconfig, pcsclite, buildGoModule, fetchFromGitHub, darwin }:
```

```suggestion
{ lib, stdenv, pkg-config, pcsclite, buildGoModule, fetchFromGitHub, PCSC }:
```

Please inerhit PCSC in top-level.nix.